index.html

```diff
@@ -213,13 +216,16 @@ <h3>What is a Verifiable Credential?</h3>
 that a physical credential is intended to represent. The addition of
 digital technologies, such as digital signatures, also make verifiable
 credentials more tamperproof and therefore more trustworthy
-than their physical counterparts. Verifiable credentials can also be
-rapidly exchanged through the Internet, making them more convenient than
-their physical counterparts when needing to establish trust at a distance.
+than their physical counterparts.
```

Comment:
I think the rapid transmission of credentials (and presentations) should not be removed as a benefit here. The word "exchanged" could be replaced with "transmitted" if that helps avoiding the notion that credentials are always shared (where we know they are not when using ZKPs).

Reply:
The statement `rapidly exchanged through the Internet, making them more convenient than
their physical counterparts` is fairly obvious so we removed it.

Reply:
Let's change to "transmitted" instead of "exchanged". And I suggest we remove "through the Internet", since this same advantage applies to bluetooth, transmission over short-wave radio, etc. In other words, it's not the Internet that's the source of the advantage--it's the digtalness. So maybe "Verifiable credentials can also be rapidly transmitted, making them more convenient than their physical counterparts when needing to establish trust at a distance."

Reply:
Updated